### DISCUSSION TITLE
Does the live woker list store in redis key named “_kombu.binding.celeryev”?

### DISCUSSION THREAD

**Discussion by Liubey** [celery/celery#7959] [Q&A]
Hi everyone
I want to monitor the worker who is alive and dead.
Does the live woker list store in redis key named “_kombu.binding.celeryev”?
If confirmed, I can read the content of the key to monitor workers😄

[a comment by an automated account is not shown]

**Comment by auvipy** [comment]
closing as duplicate of https://github.com/celery/kombu/issues/1624
